LendingAPI.md

```diff
@@ -0,0 +1,174 @@
+# Lending API
+
+## /lending/providers
```

Comment:
is there a big need to have this file in the root directory?

Reply:
No, this was just temp in this PR, sorry.  Will move it to internal.